docs/authentication-authorization/authentication/third-party-account.md

```diff
@@ -46,7 +46,13 @@ module.
    The `imageURI` property specifies the URL of the logo image. The image must be uploaded to
    the `assets/images/clients` folder before you can use this property.
 
-7. If you have already signed up, you will be directed to the Klaw home
+7. Each SSO provider can have a different "preferred username" attribute that identifies your user correctly.
+   Check your provider's documentation and select the attribute/claim that defines the username/login name.
+   Update this attribute in the `application.properties` file, replacing 'preferred_username' with their attribute.
+
+   `klaw.ad.username.attribute=preferred_username`
```

Comment:
We can update this pr abit based on the pr https://github.com/Aiven-Open/klaw/pull/2556